src/backend/distributed/utils/aggregate_utils.c

```diff
@@ -365,9 +405,11 @@ worker_partial_agg_sfunc(PG_FUNCTION_ARGS)
 
 	HeapTuple aggtuple = GetAggregateForm(box->agg, &aggform);
 	Oid aggsfunc = aggform->aggtransfn;
+	combinedInputDesc = box->combinedInputDesc;
```

Comment:
Minor note: I think the use pattern of `combinedInputDesc ` is a little hard to follow. We first use it for `lookup_rowtype_tupdesc`, later `ReleaseTupleDesc ` and then here set to `box->combinedInputDesc`. 
Could we maybe eliminate the use of `combinedInputDesc ` after here ( move its declaration where it it used with `lookup_rowtype_tupdesc `), and simply use `box->combinedInputDesc`